src/views/Auth/Login.jsx

```diff
@@ -21,28 +21,35 @@ export default function Login() {
 
     // TODO: If login was unsuccessful, set an error with a message
     // to display to the user that their login failed.
-    //
     // If login was successful, use the history hook
-    // from React Router to replace the current URL with the URL
-    // we need to redirect to.
-    // See https://v5.reactrouter.com/web/api/history for the appropriate method to use
+    
+    if(loginWasSuccessful) {
```

Comment:
Here is also a great you always want to check to make sure the user's login was a sucsess